src/validations.ts

```diff
@@ -80,8 +81,8 @@ export const validateBorrow = (body: ActionRequest): SafeActionRequest => {
 
 export const validateRepay = (body: ActionRequest): SafeActionRequest => {
   const schema = Joi.object({
-    loanId: addressSchema.optional(),
-    nfts: Joi.array().items(nftSchema).min(1).max(100).required()
+    loanId: hashSchema.optional(),
+    nfts: Joi.array().items(nftSchema).min(0).max(100).optional()
```

Comment:
What's the flow here? @Vinkersa 

can call repay with a loanId OR an array of nfts? in that case it needs .optional() not min(0). min(0) still forces you to pass an empty array of nfts. optional allows you to skip it all together

also if we have optional and min(1) it means that you can skip nfts but, if you pass nfts, you need at least one item

and building on top of this. we can use when like this:
```js
.when('loanId', {
      is: Joi.exist(),
      then: Joi.optional(),
      otherwise: Joi.required()
    })
```
so it validates that you need to pass either a loanid or an array of nfts (with min(1))

but I'm not fully sure what are requirements here